spec/requests/atmosphere/api/port_mapping_templates_spec.rb

```diff
@@ -173,7 +173,7 @@
 
     context 'when unauthenticated' do
       it 'returns 401 Unauthorized error' do
-        post api("/port_mapping_templates")
+        post api("/port_mapping_templates"), new_port_mapping_template_request
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.